core/filter_api.php

```diff
@@ -1236,7 +1236,7 @@ function filter_get_bug_rows( &$p_page_number, &$p_per_page, &$p_page_count, &$p
 
 		foreach( $t_project_ids as $t_pid ) {
 			# limit reporters to visible projects
-			if( ( ON === $t_limit_reporters ) && ( !access_has_project_level( config_get( 'report_bug_threshold', null, $t_user_id, $t_pid ) + 1, $t_pid, $t_user_id ) ) ) {
+			if( ( ON === $t_limit_reporters ) && ( !access_has_project_level( access_threshold_min_level( config_get( 'report_bug_threshold', null, $t_user_id, $t_pid ) ) + 1, $t_pid, $t_user_id ) ) ) {
```

Comment:
Yes, probably. Baically the related questions that need to be aswered usually are:
Do i have access to this bug?
Can I see all bugs or just mine?

However, after hitting the limitations of this option (which is the reason i had to use array thresholds, and find the bugs...)
I think it needs to be rethinked, into a more general "visibility" configuration.